.circleci/config.yml

```diff
@@ -1113,6 +1124,33 @@ jobs:
           # Fortunately when the artifact does not exist it just says "No artifact files found" and ignores it.
           path: all-bytecode-reports.zip
 
+  b_release_binaries:
+    <<: *base_ubuntu2004
+    steps:
+      - checkout
+      - run:
+          name: Install dependencies
+          command: |
+            apt-get update
+            apt-get install --assume-yes --no-install-recommends jq
+      - run:
+          name: Download artifacts of dependent jobs
+          command: |
+            mkdir github/
+            cd github/
+            ../scripts/solc-bin/download-circleci-binaries.sh "$CIRCLE_WORKFLOW_ID"
```

Comment:
Wouldn't it be easier, cleaner and safer to rename the binaries in the respective builds already and attach them to a common workspace similar to what we do for the bytecode run?

Reply:
Yeah, I know the problem with path conflicts in merging workspaces... but I thought since we have the same problem with the bytecode comparison runs, it may just work or at least not be too hard to achieve, but not entirely sure...